README.md

```diff
@@ -120,14 +120,14 @@ available---for instance, at the time of this writing, Ubuntu 16.04 only has
 mlpack 2.0.1 available.  Options include upgrading your Ubuntu version, finding
 a PPA or other non-official sources, or installing with a manual build.
 
-There are some other useful pages to consult in addition to this section:
+There are some useful pages to consult in addition to this section:
 
   - [Building mlpack From Source](https://www.mlpack.org/doc/mlpack-git/doxygen/build.html)
   - [Building mlpack From Source on Windows](https://www.mlpack.org/doc/mlpack-git/doxygen/build_windows.html)
 
 mlpack uses CMake as a build system and allows several flexible build
-configuration options. One can consult any of numerous CMake tutorials for
-further documentation, but this tutorial should be enough to get mlpack built
+configuration options. You can consult any of the CMake tutorials for
+further documentation, but this tutorial should be eough to get mlpack built
```

Comment:
This should be "enough" and not "eough".

Reply:
Nice catch, I missed it :)